project_config.yaml

```diff
@@ -0,0 +1,82 @@
+catalog: users # Change this to the catalog you're using
+schema: martijn_hofstra # Change this to the schema where the hotel reservations dataset resides
+table_name: hotel_reservations
+
+parameters:
+  learning_rate: 0.01
+  n_estimators: 1000
+  max_depth: 6
+
+num_features:
+  no_of_adults:
+    type: integer
```

Comment:
nice job on adding the data types and constraints!

Reply:
Thanks :) This was actually suggested by `coderabbitai`, I think it works pretty well